src/Components/Lanyard.jsx

```
@@ -0,0 +1,151 @@
+import { BallCollider, CuboidCollider, Physics, RigidBody, useRopeJoint, useSphericalJoint } from '@react-three/rapier';
+import { MeshLineGeometry, MeshLineMaterial } from 'meshline';
+
+import cardGLB from "/src/assets/lanyard/card.glb";
+import lanyard from "/src/assets/lanyard/lanyard.webp";
+
+import * as THREE from 'three';
+
+
+extend({ MeshLineGeometry, MeshLineMaterial });
+
+export default function Lanyard({ position = [0, 0, 10], gravity = [0, -40, 0], fov = 20, transparent = true }) {
```

ESLint

[error] 'position' is missing in props validation

(react/prop-types)